src/application/create_train_test_split.py

```diff
@@ -56,27 +65,53 @@ def load_patient_ids(view="FOR_kohorte_demografi_inkl_2021_feb2022"):
             new_colname=outcome,
             id_colname="dw_ek_borger",
         )
+
+        n_with_outcome = combined_df[combined_df[outcome] == 1].shape[0]
+        unsplit_outcome_props[outcome] = round(n_with_outcome / n_in_split["total"], 4)
+
         msg.good(f"Added {outcome}")
 
     val_and_test_prop = 0.3
-    val_test_split = 0.5 
+    val_test_split = 0.5
     # Meaning that the prop of the dataset that ends in val is val_test_split * val_and_test_prop (e.g. 0.3 * 0.5 = 0.15)
 
-
-    X_train, X_intermediate = train_test_split(
+    msg.good("Starting train/intermediate split")
+    X_train, X_intermediate = stratified_split_by_each_category(
```

Comment:
This is fine, but wanted to mention that python typically use `_` to denote intermediate values. E.g. `X_` (not to be confused with just `_` which is a variables not intended to be used)